melodies_monet/driver.py

```diff
@@ -1364,10 +1360,11 @@ def pair_data(self, time_interval=None):
                             
                             paired_data = sutil.omps_nm_pairing_apriori(model_obj,obs.obj,keys)
                         else:
-                            model_obj = mod.obj[keys+['dp_pa']]
+                            keys.extend('dp_pa')
+                            model_obj = mod.obj[keys]
                             paired_data = sutil.omps_nm_pairing(model_obj,obs.obj,keys)
 
-                        paired_data = paired_data.where((paired_data.o3vmr > 0))
+                        paired_data = paired_data.where((paired_data.ozone_column > 0))
```

Comment:
I had been told that, as weird as it sounds, we should not discard negative values. Might be wrong though.
Could it be left to the filter options for the user? Or is there something in the reader that makes something like that difficult?

Reply:
This part is for OMPS, where I needed to handle a bit better where there wasn't actually any obs data. From the obs files, there is a valid range of [ 50. 700.] for the total column O3. 
How about a discard of where the obs is np.nan?

```suggestion
                        paired_data = paired_data.where(~np.isnan(paired_data.ozone_column))
```

Reply:
Looks good to me. If you think that keeping the `>0` option is better, I don't have a strong opinion anyway